components/core/SearchDropdown.js

```diff
@@ -32,8 +25,10 @@ const STYLES_DROPDOWN_ITEM = css`
   font-size: 0.8em;
   border-radius: 16px;
   border: 1px solid ${Constants.system.white};
+  cursor: pointer;
+
   :hover {
-    border-color: ${Constants.system.border};
+    border-color: ${Constants.system.border} !important;
```

Comment:
It's to override an in-line style that sets it depending on whether it's highlighted using arrow keys or not (since the user can highlight using mouse or arrow keys). I tried taking it out but then it doesn't highlight on hover. And if I set the inline style to 
```
style={{
  borderColor:
    this.state.selectedIndex === i
      ? Constants.system.border
      : "auto",
  }}
```
rather than 
```
style={{
  borderColor:
    this.state.selectedIndex === i
      ? Constants.system.border
      : Constants.system.white,
  }}
```
then things stay highlighted after they've been highlighted with using the arrow keys and don't revert back once you navigate away.